pkg/services/datasources/service/datasource.go

```diff
@@ -680,7 +680,7 @@ func (s *Service) getCustomHeaders(jsonData *simplejson.Json, decryptedValues ma
 		}
 
 		if val, ok := decryptedValues[headerValueSuffix]; ok {
-			headers[key] = val
+			headers.Add(key, val)
```

Comment:
Are we sure we want to `Add` here and not `Set`?

Reply:
Yes, if not, only the last value set in the SecureJSONData will be set.

Reply:
@andresmgot This is the goal of this: allow to set the same header several time. See https://github.com/grafana/grafana-plugin-sdk-go/issues/829

Reply:
That's why we need to use `.Add`, not `.Set`, right? Unless I am missing something.

Reply:
Thought so - would be good to add a test here in that case.

Reply:
:+1: added a test to cover this